tests/core/test_em_js.cpp

```diff
@@ -60,15 +60,23 @@ EM_JS(char*, return_utf8_str, (void), {
     var lengthBytes = lengthBytesUTF8(jsString)+1;
     var stringOnWasmHeap = _malloc(lengthBytes);
     stringToUTF8(jsString, stringOnWasmHeap, lengthBytes);
+#if __wasm64__
+    return BigInt(stringOnWasmHeap);
```

Comment:
It seems a little odd that this is needed, since this value arrives from wasm on line 61. That is, a cast is needed just to roundtrip here... should line 61 perhaps automatically return a BigInt, or should we have an automatic cast on this side?